src/FastRandom.cpp

```diff
@@ -8,8 +8,11 @@
 
 #include <random>
 
-#if defined (__GNUC__)
-	#define ATTRIBUTE_TLS static __thread
+#if defined (ANDROID)
+	// Tycho: I'm sorry, I'm sorry, I'm so, so, sorry, but please blame the NDK developers for lack of, thread_local
+	#define ATTRIBUTE_TLS static
```

Comment:
They are working on it. Unless someone is actually really in need of running a reliable server on their phone, it doesn't matter in my humble opinion.